dashboard/test/controllers/api/v1/pd/workshops_controller_test.rb

```diff
@@ -44,15 +44,21 @@ class Api::V1::Pd::WorkshopsControllerTest < ::ActionController::TestCase
     assert_equal workshop_2.id, response[0]['id']
   end
 
-  test 'with the enrollments param, enrollment IDs are included in the response' do
-    sign_in @organizer
-    enrollment = create :pd_enrollment, workshop: @workshop, email: @organizer.email
+  test 'workshops_user_enrolled_in returns workshops the user is enrolled in' do
+    teacher = create :teacher
+    sign_in(teacher)
+
+    workshop_2 = create :pd_workshop
+
+    enrollment_1 = create(:pd_enrollment, workshop: @workshop, email: teacher.email, user_id: nil)
+    enrollment_2 = create(:pd_enrollment, workshop: workshop_2, email: 'other@example.com', user_id: teacher.id)
+
+    get :workshops_user_enrolled_in
 
-    get :index, params: {include_enrollments: true}
-    assert_response :success
     response = JSON.parse(@response.body)
-    assert_equal @workshop.id, response[0]['id']
-    assert_equal enrollment.id, response[0]['enrollment_id']
+    assert_equal 2, response.length
+    assert_equal enrollment_1.code, response.find { |workshop| @workshop.id == workshop['id']}['enrollment_code']
+    assert_equal enrollment_2.code, response.find { |workshop| workshop_2.id == workshop['id']}['enrollment_code']
```

Comment:
Add an additional workshop with an enrollment from another teacher (that is not returned from the API), to make sure that this only returns the workshops enrolled in by the logged in user.